compiler/rustc_target/src/spec/aarch64_unknown_teeos.rs

```diff
@@ -0,0 +1,20 @@
+use crate::spec::Target;
+
+use super::{Cc, LinkerFlavor, Lld};
+
+pub fn target() -> Target {
+    let mut base = super::teeos_base::opts();
+    base.features = "+strict-align,+neon,+fp-armv8".into();
+    base.max_atomic_width = Some(128);
+    base.linker_flavor = LinkerFlavor::Gnu(Cc::Yes, Lld::No);
+    base.linker = Some("rust-lld".into());
```

Comment:
`linker` and `linker_flavor` are not architecture-specific and can be moved to `teeos_base.rs`.